src/Tests/ObjectLifetimeTests/Directory.Build.props

```diff
@@ -0,0 +1,2 @@
+<Project>
```

Comment:
Are these empty files needed? Is this overriding default files we don't want?

Reply:
Yes, it's to prevent inheriting items we don't want.

Reply:
Worh adding a comment to clarify that?